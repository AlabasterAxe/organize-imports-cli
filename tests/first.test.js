const runCli = require("./run-cli");

describe("The first Test", () => {
  runCli("cli/first", ["file1.js"]).test();
});
