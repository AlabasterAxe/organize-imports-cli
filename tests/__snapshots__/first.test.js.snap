// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The first Test (status) 1`] = `0`;

exports[`The first Test (stderr) 1`] = `""`;

exports[`The first Test (stdout) 1`] = `
"Organizing imports...
<dir>/file1.js/*CR*/
<dir>/file1.js (modified)
Done!
"
`;

exports[`The first Test (write) 1`] = `
Array [
  Object {
    "content": "import path from \\"path\\";

console.log(path.resolve(\\".\\"));
",
    "filename": "file1.js",
  },
]
`;
